gpt2.py

```diff
@@ -0,0 +1,45 @@
+import gpt_2_simple as gpt2
```

Comment:
@ddong63 were you able to get this to run locally? I've tried running the file in the docker container and I think we need to add gpt_2_simple to our docker image because i'm getting a `No module named 'gpt_2_simple` error.

Reply:
agreed! I was running gpt_2_simple locally. This definitely needs to be added to our docker image.

Reply:
@briannabrownOE I added gpt2 to docker (https://github.com/WomenInDataScience-Seattle/FortuneCookie/pull/15/commits/45910ff130907f6008f855c6d445431c79dfa1da). Can you give it a try and see if it works? Thank you!